lib/zuora_client.rb

```diff
@@ -122,51 +122,46 @@ def query(query_string)
 
     def subscribe(obj)
       begin
-        response = @client.api_call(:subscribe, obj)
-        return response
+        @client.api_call(:subscribe, obj)
       rescue Exception => e
-        puts e.message
+        [errors: {message: e.message, code: nil}, object: obj]
```

Comment:
Since this doesn't do the same hashing, we can't output something similar to what would normally be output.